src/components/RichText.ts

```diff
@@ -148,7 +150,33 @@ export class RichText extends Component<RichTextProps> {
     private setUpEditor(props: RichTextProps) {
         if (this.quillNode && !this.quill) {
             this.quill = new Quill(this.quillNode, {
-                modules: this.getEditorOptions(),
+                modules: {
+                    ...this.getEditorOptions(),
+                    keyboard: this.props.onTabKey === "changeFocus"
+                        ? {
+                            bindings: {
+                                indent: {
+                                    key: "Tab",
+                                    format: [ "blockquote", "indent", "list" ],
+                                    handler: (_range, context) => {
```

Comment:
I prefer you make the function a named function. Most cases anonymous functions consume memory resources.

Reply:
Optimised